aiida_fleur/workflows/scf.py

```diff
@@ -524,12 +531,15 @@ def get_res(self):
         mode = self.ctx.wf_dict.get('mode')
         if self.ctx.parse_last:
             last_calc = self.ctx.last_calc
-
+            print(last_calc)
             # TODO: dangerous, can fail, error catching
             # TODO: is there a way to use a standard parser?
             out_para = last_calc.outputs.output_parameters
-            fleur_calcjob = load_node(out_para.get_dict()['CalcJob_uuid'])
-            outxmlfile_opened = last_calc.outputs.retrieved.open(
+            fleur_calcjob = out_para.get_incoming().all()[-1].node
```

Comment:
Usually I prefer more explicit function that searches for the last CalcJob submitted by a WorkChain.
For example, see aiida_fleur.tools.common_fleur_wf.find_last_in_restart(). 
Is it guaranteed that the last incoming node will be the last CalcJob?